bundle/src/main/java/com/adobe/acs/commons/version/package-info.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Version services and helpers.
  */
-@Version("1.0.0")
+@Version("2.0.0")
```

Comment:
This looks wrong -- we shouldn't need to increment the major version. I suspect that EvolutionContext is missing the `@ProviderType` annotation.

Reply:
Oh, good catch!   Yeah, @ProviderType is definitely a good idea, because then adding a method would not require a major version increment like this.

Reply:
I'll check it now, without major version update and without @providertype there is a en error from maven baseline plugin -> version increase required 1.0.0 -> 2.0.0